src/Resources/public/js/pimcore/startup.js

```diff
@@ -1,17 +1,72 @@
-pimcore.registerNS("pimcore.plugin.YouweDataDictionaryBundle");
+pimcore.registerNS("pimcore.plugin.dataDictionaryBundle");
 
-pimcore.plugin.YouweDataDictionaryBundle = Class.create(pimcore.plugin.admin, {
+pimcore.plugin.dataDictionaryBundle = Class.create(pimcore.plugin.admin, {
     getClassName: function () {
-        return "pimcore.plugin.YouweDataDictionaryBundle";
+        return "pimcore.plugin.dataDictionaryBundle";
     },
 
     initialize: function () {
         pimcore.plugin.broker.registerPlugin(this);
     },
 
     pimcoreReady: function (params, broker) {
-        // alert("YouweDataDictionaryBundle ready!");
+        var user = pimcore.globalmanager.get('user');
+
+        if (user.isAllowed('data_flow_permission')) {
+
+            //Add item to settings menu
+            var importMenu = new Ext.Action({
+                text: t('Show current dataflow'),
```

Comment:
```suggestion
                text: t('Show Data Dictionary'),
```
The name of the package is *Data Dictionary*. All the text references should mention *Data Dictionary*. The data model is just one part of it (the first one).

Reply:
I agree but you show the flow here, the rest of the documentation is at the moment not there.